moonstreamapi/moonstreamapi/routes/subscriptions.py

```diff
@@ -559,3 +566,158 @@ async def list_subscription_types() -> data.SubscriptionTypesListResponse:
         raise MoonstreamHTTPException(status_code=500, internal_error=e)
 
     return data.SubscriptionTypesListResponse(subscription_types=results)
+
+
+@router.get(
+    "/is_contract",
+    tags=["subscriptions"],
+    response_model=data.ContractInfoResponse,
+)
+async def address_info(request: Request, address: str):
+    """
+    Looking if address is contract
+    """
+
+    user_token = request.state.token
+
+    try:
+        Web3.toChecksumAddress(address)
```

Comment:
I suggest to remove here `try..catch` block and use this pattern at `ContractInfoResponse` https://github.com/moonstream-to/api/blob/d3b2e350f522879dfe6658ac0d843010b8be70c8/engineapi/engineapi/data.py#L276